src/monitoring/prometheus.rs

```diff
@@ -91,6 +91,31 @@ lazy_static! {
         "Total number of error logs emitted by node"
     )).unwrap();
 
+    pub static ref LAST_EXECUTION_READ_COUNT: IntGauge = register_int_gauge!(opts!(
+        "execution_cost_read_count",
```

Comment:
I think these could be slightly more descriptively named for prometheus as `last_block_read_count`, etc.

Reply:
In the variable name or the caption part?

Reply:
In the prometheus metric name, i.e.

```rust
    pub static ref LAST_EXECUTION_READ_COUNT: IntGauge = register_int_gauge!(opts!(
        "last_block_read_count",
```

Reply:
Can we also prepend `stacks_node_` to each metric name like we do for the other stacks-node metrics? It helps with search and discovery, especially when your Prometheus deployment could have hundreds of metrics from all kinds of services.
![Screen Shot 2022-01-31 at 3 24 23 PM](https://user-images.githubusercontent.com/2747302/151867563-4ceb3565-6956-45dc-8b3f-c84361cf3b2a.png)

Reply:
Went with `stacks_node_last_block_read_count`-style prefix.